lib/googleauth/compute_engine.rb

```diff
@@ -59,20 +59,15 @@ class << self
         extend Memoist
 
         # Detect if this appear to be a GCE instance, by checking if metadata
-        # is available
+        # is available.
+        #
+        # TODO: This should use google-cloud-env.
```

Comment:
Do you want to move this comment inside the method body? So it isn't used as documentation?

Reply:
Yeah good call. Moved.